lib/Mojo/Util.pm

```diff
@@ -23,6 +23,11 @@ use Unicode::Normalize ();
 # Check for monotonic clock support
 use constant MONOTONIC => eval { !!Time::HiRes::clock_gettime(Time::HiRes::CLOCK_MONOTONIC()) };
 
+# For speeding up @{ $foo // [] }
+use constant _empty => [];
```

Comment:
We don't use lowercase constants.

Reply:
Something like `_EMPTY_ARRAY` would also be more descriptive.

Reply:
Fixed.